ember-phone-input/src/components/phone-input.hbs

```diff
@@ -0,0 +1,12 @@
+<input
+  type={{this.type}}
+  disabled={{this.disabled}}
+  required={{this.required}}
+  autocomplete={{this.autocomplete}}
+  {{on 'input' this.onInput}}
+  {{did-insert this.onDidInsert}}
+  {{did-update this.onDidUpdate @country @number}}
+  {{will-destroy this.onDestroy}}
+  data-test-loading-iti={{this.loadingIti}}
+  ...attributes
```

Comment:
question: by spreading attributes in this way, can the user override some attributes in an unwanted way? my point is that we don't want some attributes to be overridden (e.g.: `type`, `disabled` or `required`)

Reply:
All attributes are already accepted as args, and also I think this makes the input very configurable which should be a positive.

Reply:
Yeah I understand that it brings good value in general, I was asking myself if we're ok with the user overriding things like `type`. I modified my message above, because I understand that we may want `disabled` and `required` to be configurable. I'm just not sure about `type` for instance, what do you think?

Reply:
What do you think would be the negative point of changing `type`? As I see it the ability to change as much as possible helps with edge cases where possibly we'll need a `type` number. I know I've been in situations where libraries have very limited configurability and it ended up being a problem.

Reply:
This is a phone input component, so from my point of view, types other than `text` or maybe `number` wouldn't make sense here.

What I'm asking myself is: would `button`, `file` or `email` make sense as `type` here?

Anyway, maybe we're enforcing this in some other way and I'm missing something. Semantically speaking I think that we should allow a limit set of types here.

Reply:
Ah ok I understand. But in that case we would just add assertions for limitations within the component. Or do you think I should just move attributes extension to the top so they can't be overriden?

Reply:
I think that we can move to the bottom the attributes that we don't want to be overriden, as `type`.

Also by looking at the doc I see that changing `type` is not documented (and it's the right move): https://qonto.github.io/ember-phone-input/versions/master/docs/api/components/phone-input

Reply:
Ok, fixed!